dependency/del.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```

Comment:
What is this script for?

Reply:
Its a test script that got committed by accident. I've deleted this.